### DISCUSSION TITLE
搭建开发环境启动报错

### DISCUSSION THREAD

**Discussion by zhuangyong666** [whyour/qinglong#1725] [Q&A]
![image](https://user-images.githubusercontent.com/18116324/203575698-69705044-2e18-464d-a0b7-83a7f82e0a21.png)
![image](https://user-images.githubusercontent.com/18116324/203576036-82e45bcc-b2c5-4959-8903-63f3bf57ad53.png)

**Comment by 1947839940** [comment]
> ![image](https://user-images.githubusercontent.com/18116324/203575698-69705044-2e18-464d-a0b7-83a7f82e0a21.png)
> 
> ![image](https://user-images.githubusercontent.com/18116324/203576036-82e45bcc-b2c5-4959-8903-63f3bf57ad53.png)
> 
> 

请问你开发环境报错是如何解决的？我目前发现在windows下有这个问题，mac，Linux下没问题
